kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/impl/SharedInformerFactoryImpl.java

```diff
@@ -106,11 +107,11 @@ public synchronized <T> SharedIndexInformer<T> getExistingSharedIndexInformer(Cl
 
   @Override
   public synchronized Future<Void> startAllRegisteredInformers() {
-    List<CompletableFuture<Void>> startInformerTasks = new ArrayList<>();
+    List<CompletionStage<Void>> startInformerTasks = new ArrayList<>();
```

Comment:
This doesn't seem to be compliant with line 128.
I'm going to add a subsequent commit to ensure this won't cause any issue (there shouldn't be any issue because the underlying impl will be CompletableFuture, but the code is wrong nonetheless).